Makefile

```diff
@@ -113,7 +134,82 @@ preload-images:
 
 validate: pkg/subctl/operator/common/embeddedyamls/yamls.go
 
-.PHONY: test validate build ci clean generate-clientset generate-embeddedyamls generate-operator-api operator-image preload-images
+all: manager
+
+# Run tests
+#test: generate fmt vet manifests
+	#go test ./... -coverprofile cover.out
+
+# Build manager binary
+manager: generate fmt vet
+# We check BUILD_ARGS since that's what the compile script uses
+ifeq (--debug,$(findstring --debug,$(BUILD_ARGS)))
+	go build -o bin/submariner-operator main.go
+else
+	go build -ldflags -s -w -o bin/submariner-operator main.go
+endif
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet manifests
+	go run ./main.go
```

Comment:
Where does this come from, we don't need it in our case have our own E2E testing

Reply:
oh yes, I can see it in shipyard we can remove `fmt` and `vet`
`make manager` builds the operator binary using `go build` since in operator-sdk v1 there's no `build` command anymore.
`make run` runs the operator on a k8s cluster during development it can be removed as well